core/job/service/job_service.go

```diff
@@ -913,6 +913,7 @@ func (j *JobService) generateJobs(ctx context.Context, tenantWithDetails *tenant
 }
 
 func (j *JobService) compileConfigs(configs job.Config, tnnt *tenant.WithDetails) map[string]string {
+	const projectConfigPrefix = "GLOBAL__"
```

Comment:
Can this be moved to a const ?

Reply:
@sbchaos , this is not being used anywhere. but we could move it to the top